numpy/lib/function_base.py

```diff
@@ -2965,7 +2965,7 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
 
     >>> x = [-2.1, -1,  4.3]
     >>> y = [3,  1.1,  0.12]
-    >>> X = np.vstack((x,y))
+    >>> X = np.concatenate((x,y), axis=0)
```

Comment:
Not the same for 1D arrays, they need to be reshaped, which is what `stack` does.